<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>588</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Lower bound of integral</string>
         </property>
         <property name="text">
          <string>From</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IntegLower">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Lower bound of integral</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="IntegCalc">
       <property name="toolTip">
        <string>Calculate the integral - Shift + Enter</string>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="IntegOutType">
         <property name="toolTip">
          <string>Format of output</string>
         </property>
         <property name="text">
          <string>Output Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="IntegPP">
         <property name="toolTip">
          <string>Pretty print</string>
         </property>
         <property name="text">
          <string>Pretty</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="IntegLatex">
         <property name="toolTip">
          <string>Latex</string>
         </property>
         <property name="text">
          <string>Latex</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="IntegNormal">
         <property name="toolTip">
          <string>Normal - no special format</string>
         </property>
         <property name="text">
          <string>Normal</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="IntegPrev">
       <property name="toolTip">
        <string>Preview the integral</string>
       </property>
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QTextEdit" name="IntegExp">
       <property name="toolTip">
        <string>Expression</string>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="placeholderText">
        <string>Expression</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Upper bound of integral</string>
         </property>
         <property name="text">
          <string>To</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IntegUpper">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Upper bound of integral</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Integrate with respect to variable</string>
         </property>
         <property name="text">
          <string>Variable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IntegVar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Integrate with respect to variable</string>
         </property>
         <property name="text">
          <string>x</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1" rowspan="4">
      <widget class="QTextBrowser" name="IntegOut">
       <property name="toolTip">
        <string>Exact answer of calculation</string>
       </property>
       <property name="placeholderText">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" rowspan="3">
      <widget class="QTextBrowser" name="IntegApprox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Approximate answer of calculation</string>
       </property>
       <property name="placeholderText">
        <string>Approximate Answer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
